### PR TITLE
Adds support for commenting on PostgreSQL

### PR DESCRIPTION
**Pull request by eniolopes**
TODO:

- [x] Support comments when doing changes to tables and columns (ALTER TABLE)
- [x] Support comments on constraints and anything else supported by us on [this list](https://www.postgresql.org/docs/9.5/static/sql-comment.html)

### CODE REVIEW

**Comment by eniolopes** [comment]
@josevalim any ideas on why this would be falling? locally tests are all green

File: lib/ecto/adapters/postgres/connection.ex [hunk 1]
```diff
@@ -803,6 +829,8 @@ if Code.ensure_loaded?(Postgrex) do
       <<?", name::binary, ?">>
     end
 
+    defp single_quote(value), do: "'#{value}'"
```
Comment (josevalim) [inline lib/ecto/adapters/postgres/connection.ex]: Shouldn't we also escape any quote inside `value`?

File: lib/ecto/adapters/postgres/connection.ex [hunk 2]
```diff
@@ -626,6 +633,25 @@ if Code.ensure_loaded?(Postgrex) do
       end
     end
 
+    defp comment_on(_database_object, _name, nil), do:  ""
+    defp comment_on(:column, {table_name, column_name}, comment) do
+      column_name = quote_table(table_name, column_name)
+      "; COMMENT ON COLUMN #{column_name} IS #{single_quote(comment)}"
+    end
+
+    defp comment_on(database_object, name, comment) do
```
Comment (josevalim) [inline lib/ecto/adapters/postgres/connection.ex]: Instead of doing Atom.to_string |> String.upcase, we should explicitly list everything we support. :)

**Comment by josevalim** [comment]
The build failures are unrelated, so please go ahead!

**Comment by eniolopes** [comment]
@josevalim please advise if commenting should be supported when renaming tables and columns, I think these are the only ones that I didn't add support for it.

Also, should anything be added to the integration tests specifically for this? If so, please advise how it should be done.

I want to be able to finish this today. Thanks for the support and opportunity to contribute ❤️

**Comment by eniolopes** [comment]
This close https://github.com/elixir-ecto/ecto/issues/1531

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**